src/main/java/io/phasetwo/service/datastore/KeycloakOrgsExportImportManager.java

```diff
@@ -0,0 +1,270 @@
+package io.phasetwo.service.datastore;
+
+import static io.phasetwo.service.Orgs.ORG_OWNER_CONFIG_KEY;
+import static org.keycloak.models.utils.StripSecretsUtils.stripForExport;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.phasetwo.service.datastore.representation.KeycloakOrgsRealmRepresentation;
+import io.phasetwo.service.datastore.representation.OrganizationAttributes;
+import io.phasetwo.service.datastore.representation.OrganizationRepresentation;
+import io.phasetwo.service.datastore.representation.OrganizationRoleRepresentation;
+import io.phasetwo.service.model.InvitationModel;
+import io.phasetwo.service.model.OrganizationModel;
+import io.phasetwo.service.model.OrganizationProvider;
+import io.phasetwo.service.resource.Converters;
+import io.phasetwo.service.resource.OrganizationAdminAuth;
+import jakarta.ws.rs.NotAuthorizedException;
+import jakarta.ws.rs.core.HttpHeaders;
+import jakarta.ws.rs.core.MediaType;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import lombok.extern.jbosslog.JBossLog;
+import org.jboss.logging.Logger;
+import org.keycloak.common.util.CollectionUtil;
+import org.keycloak.exportimport.ExportAdapter;
+import org.keycloak.exportimport.ExportOptions;
+import org.keycloak.exportimport.util.ExportUtils;
+import org.keycloak.jose.jws.JWSInput;
+import org.keycloak.jose.jws.JWSInputException;
+import org.keycloak.models.Constants;
+import org.keycloak.models.IdentityProviderModel;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.ModelException;
+import org.keycloak.models.RealmModel;
+import org.keycloak.models.utils.KeycloakModelUtils;
+import org.keycloak.representations.AccessToken;
+import org.keycloak.representations.idm.RealmRepresentation;
+import org.keycloak.services.managers.AppAuthManager;
+import org.keycloak.services.managers.AuthenticationManager;
+import org.keycloak.services.managers.RealmManager;
+import org.keycloak.services.resources.admin.AdminAuth;
+import org.keycloak.storage.ImportRealmFromRepresentationEvent;
+import org.keycloak.storage.datastore.DefaultExportImportManager;
+import org.keycloak.util.JsonSerialization;
+
+@JBossLog
+public class KeycloakOrgsExportImportManager extends DefaultExportImportManager {
+  private final KeycloakSession session;
+  private final AdminAuth auth;
+  private static final Logger logger = Logger.getLogger(KeycloakOrgsExportImportManager.class);
```

Comment:
this is unnecessary. the `@JBossLog` annotation gives you a static `log` Logger already.